.gitignore

```diff
@@ -396,3 +396,7 @@ gcov.css
 
 # Generated by unit tests files
 tests/data/*.translation
+
+# Godot-cpp headers
+modules/text_server_adv/gdextension_build/godot-cpp
+modules/text_server_fb/gdextension_build/godot-cpp
```

Comment:
These should likely go in a dedicated `.gitignore` in the modules themselves, to stay self-contained.

Reply:
Moved to modules.